<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_window</class>
 <widget class="QDialog" name="login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно авторизации</string>
  </property>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> Логин:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Пароль:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_login">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_password">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(46, 174, 46);
border: 2px solid black;</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(126, 145, 255);
border: 2px solid black;</string>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>или</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
